src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -309,14 +251,13 @@
               :column-data column-data
               :sort-by (zipmap access-levels (range)) :sort-initial :asc
               :render (fn [data]
-                        (let [access-level (:access-level data)]
+                        (let [access-level (:access-level data)
+                              workspace-id (:workspace-id data)
+                              ws-auth-domains (:auth-domains data)]
```

Comment:
You can do a map destructure here:
```clojure
(let [{:keys [access-level workspace-id auth-domains]} data] ... )
```
(And then use `auth-domains` below of course)